e2e/pinned_dependencies_test.go

```diff
@@ -49,9 +49,9 @@ var _ = Describe("E2E TEST:"+checks.CheckPinnedDependencies, func() {
 			}
 			expected := scut.TestReturn{
 				Error:         nil,
-				Score:         4,
+				Score:         2,
 				NumberOfWarn:  139,
-				NumberOfInfo:  3,
+				NumberOfInfo:  0,
```

Comment:
This is a placeholder for me to figure out the 3 info messages that went away.

Reply:
The repo being tested, `ossf-tests/scorecard-check-pinned-dependencies-e2e`, has:
- 0 GHA Third-party actions
- 1+ GHA GitHub-owned actions, all unpinned
- 0 npm installs
- 1+ go installs, all pinned
- and 1+ other dependencies types, all unpinned

The total Info logs decreased because previously we had logs for:
- "all Third-party actions are pinned"
- "all npm installs are pinned"
- "all go installs are pinned"
These "all pinned" Info logs were removed. Now, we only have Warn logs for "unpinned at XYZ snippet" and Debug logs for parsing errors.

Reply:
Adding some explanation here, I remove the "all pinned" Info logs, since now the proportion or pinned vs total is what matters, not only if all dependencies were pinned or not.

Reply:
https://github.com/ossf/scorecard/pull/3436#issuecomment-1705687150